monix-reactive/shared/src/main/scala/monix/reactive/observables/CombineObservable.scala

```diff
@@ -21,9 +21,9 @@ import cats.Apply
 import monix.execution.internal.Newtype1
 import monix.reactive.Observable
 
-/** A `CombineObservable` is an observable that wraps a regular
-  * [[Observable]] and provide [[cats.Apply]] instance
-  * which uses [[Observable.combineLatest]] to combine elements.
+/** Newtype encoding for an [[Observable]] datatype that has a [[cats.Apply]]
+  * instance which uses [[Observable.combineLatest]] to combine elements
```

Comment:
Do a `unidoc` command in `sbt`, make sure these references are resolved - I know that it has problems with resolving imports, so if you're not in the same package you have to do the full path, but check first:

```
[[monix.reactive.Observable.combineLatest Observable.combineLatest]]
```

Reply:
Done :+: